charts/pact-broker/templates/serviceaccount.yaml

```diff
@@ -16,4 +16,7 @@ metadata:
     {{- end }}
   {{- end }}
 automountServiceAccountToken: {{ .Values.serviceAccount.automountServiceAccountToken }}
+{{- if .Values.serviceAcccount.imagePullSecrets }}
+imagePullSecrets:
+  - name: {{ .Values.serviceAccount.imagePullSecrets }}
```

Comment:
As the `imagePullSecrets` is an array, it's best if we allow users to specify a list of pull secrets instead of only the one.